src/DBFTPlugin/Messages/PrepareRequest.cs

```diff
@@ -19,13 +19,17 @@ public class PrepareRequest : ConsensusMessage
     {
         public uint Version;
         public UInt256 PrevHash;
+        // The execution result hash of the previous block transactions
+        // since we do not need to update it or search it, no merkel tree is needed
```

Comment:
This is correct, in this case, it is not a storage, it is a protocol communication level that ensure nodes will not proceed.